.travis.yml

```diff
@@ -13,22 +13,42 @@ env:
   - ROS_DISTRO=dashing ROS_VERSION=2 GAZEBO_VERSION=9 WORKSPACES="robot_ws"
   - ROS_DISTRO=dashing ROS_VERSION=2 GAZEBO_VERSION=9 WORKSPACES="simulation_ws"
```

Comment:
I didn't see the review for this, but why are we splitting out the robot_ws and simulation_ws builds?

Reply:
Many dependencies are pulled and built from source rather than via apt-get because they were not bloomed yet. We split it to avoid hitting the 50 minutes Travis limit.